pipeline/emailing.py

```diff
@@ -10,34 +10,52 @@
 from load import create_connection
 
 
-def send_email(plant_id: int, value: float, value_type: str) -> None:
-    """Sends an email to the correct botanist if the plants conditions are worrying."""
+def send_email(plant_id: int, value: float, value_type: str, condition:str ) -> None:
+    """
+    Sends an email to the correct botanist if the plants conditions are worrying.
+    Example inputs - plant_id: 1, value: 10000, value_type: 'temperature', condition: 'exceeded'.
+    """
+
     load_dotenv()
-    html = generate_html(plant_id, value, value_type)
+    html = generate_html(plant_id, value, value_type, condition)
     client = boto3.client("ses", region_name="eu-west-2")
     message = MIMEMultipart()
-    message["Subject"] = "Plant ALERT!"
+    message["Subject"] = f"PLANT ALERT! ACCEPTABLE {value_type.upper()} LEVEL {condition.upper()}"
```

Comment:
Good subject

Reply:
Surely you mean unacceptable?

Reply:
The email title would look like "PLANT ALERT! ACCEPTABLE (TEMPERATURE / SOIL MOISTURE) EXCEEDED / NOT MET".  Unacceptable temperature or unacceptable soil moisture seemed too general.